webrtc.html

```diff
@@ -4946,10 +4947,57 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <code>degradationPreference</code> indicates which is
               preferred.</p>
             </dd>
+            <dt><dfn><code>scalingPolicy</code></dfn> of type
+            <span class="idlMemberType"><a>RTCRtpScalingPolicyType</a></span>,
+            defaulting to <code>"down-only"</code></dt>
+           <dd>
+	     <p> Controls certain aspects of how the browser
+	     can manipulate the media from
+	     a camera or microphone that is sent over RTP by this
+	     <code><a>RtpSender</a></code>.
+	     The aspects this controls are: </p>
+	     <ul>
+	       <li>
+	       <p>changing the width and height of the video frames, </p>
+	       </li>
+	       <li>
+	       <p>changing the same rate of audio, </p>
+	       </li>
+	       <li>
+	       <p>changing the same size of audio. </p>
```

Comment:
s/same/sample/ ?

Further: I did not find any SDP attribute mentioned in JSEP about audio sample size, so it seems there would be no reason for the UA to ever upscale that in any way.